packages/neuron-ui/src/components/SpecialAssetList/index.tsx

```diff
@@ -217,16 +219,17 @@ const SpecialAssetList = () => {
     (item: SpecialAssetCell) => {
       const { timestamp, customizedAssetInfo, capacity, lock, type, data } = item
 
-      return useGetSpecialAssetColumnInfo({
+      return getSpecialAssetColumnInfo({
```

Comment:
`getSpecialAssetColumnInfo` is not a hook function, could we move 
```typescript
useCallback(
    (item: SpecialAssetCell) => {
      const { timestamp, customizedAssetInfo, capacity, lock, type, data } = item

      return getSpecialAssetColumnInfo({
        cell: { capacity, lock, type, data },
        datetime: +timestamp,
        epoch,
        assetInfo: customizedAssetInfo,
        bestKnownBlockTimestamp,
        tokenInfoList,
      })
    },
    [epoch, bestKnownBlockTimestamp, tokenInfoList, useGetSpecialAssetColumnInfo]
```

to hooks file?